web/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,248 @@
+import Link from "next/link";
+import { SetStateAction, useState } from "react";
+import { MdAccountCircle, MdAddCircle } from "react-icons/md";
+
+import Button from "../../components/Inputs/Button";
+import LinkIconButton from "../../components/Inputs/LinkIconButton";
+import SearchBar from "../../components/Inputs/SearchBar";
+import ImageCard from "../../components/Surfaces/ImageCard";
+
+/*
+ *
+ * Note to viewer:
+ * Please bully, I must learn the error in my ways.
+ *
+ * */
+
+export default function Index() {
+  const [text, setText] = useState("");
+
+  function handleChange(value: string) {
+    setText(value);
+  }
+
+  const user_views = 1;
+  const user_saves = mockData.length;
+  const user_name = "username";
+  const user_icon = "https://avatars.githubusercontent.com/u/79925808?v=4";
+
+  return (
+    <>
+      {/* TOP BORDER */}
+      <div className="p-4"></div>
+
+      {/* PROFILE PICTURE */}
+      <div className="container mx-auto">
+        {/* START PHOTO PADDING */}
+        <div className="p-4"></div>
+        {/* FLEX-SEAL (wrap*) */}
+        <div className="flex flex-wrap items-center justify-center">
+          <a href="#" className="relative block">
+            {/* PROFILE PICTURE DATA */}
+            <img
+              className="custom-position h-24 w-24 rounded-full object-cover"
+              src={user_icon}
+              alt="name"
+            />
+          </a>
+        </div>
+        {/* END PHOTO PADDING */}
+        <div className="p-4"></div>
+      </div>
+
+      {/* USERNAME */}
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-center">
+          <div className="text-lg">@{user_name}</div>
+        </div>
+        <div className="p-1"></div>
+      </div>
+
+      {/* VIEWS / SAVED */}
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-center gap-2">
+          <div className="text-sm">{user_views} views</div>
+          <div className="text-sm">{user_saves} saves</div>
+        </div>
+        <div className="p-2"></div>
+      </div>
+
+      {/* CREATED / COLLECTIONS */}
+      <div className="container mx-auto">
+        <div className="p-2"></div>
+        <div className="flex flex-wrap items-center justify-center gap-2">
+          <div className="p-2">
+            <section className={"flex gap-4"}>
+              <div className={"flex flex-col items-center gap-2"}>
```

Comment:
Raciel likes to put braces around the classes, even if they aren't needed. I don't. However, I think we would both agree that you should at least be consistent.